app/javascript/application.js

```diff
@@ -1,3 +1,17 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+// import "@hotwired/turbo-rails"
+// import "./controllers"
+//= require_tree .
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './store';
+import App from './components/App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+    <Provider store={store}>
+        <App />
+    </Provider>
+);
```

Comment:
- [ ] When I start the rails server and go to the home page no greeting appeared on the page and there is an error raised on the console, Please fix that.

![image](https://github.com/Thoth1111/Hello-Rails-React/assets/68796232/78a89a88-4ab0-4503-876f-ec81ee9a10de)

Reply:
Please note that the home page is the root page so once the server is loaded, it should display the greeting. The error may have been related to the fact that there was no seed data but I've included that in the latest commit. 
Thanks.